core/src/components/input/input.scss

```diff
@@ -434,9 +455,7 @@
  * Label is on the left of the input in LTR and
  * on the right in RTL. Label also has a fixed width.
  */
-:host(.input-label-placement-fixed) label {
-  @include margin(0, 0, 0, 0);
```

Comment:
This style was wrong as it caused the input to appear right up against overflowing label text: 
![image](https://user-images.githubusercontent.com/2721089/200683618-2f715aa6-efc0-4dc4-9edc-2b5fc2ca1b21.png)


Instead, now there is 8px of spacing: 
![image](https://user-images.githubusercontent.com/2721089/200683650-a34780b8-3938-41d6-ac34-44b8c10b0c03.png)